Cargo.toml

```diff
@@ -1,16 +1,11 @@
-[package]
-name = "objpoke"
-version = "0.3.0"
-authors = ["tux3 <barrdetwix@gmail.com>"]
-edition = "2018"
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/tux3/objpoke/"
-categories = ["command-line-utilities", "development-tools::build-utils"]
-description = "Minimal in-place objcopy replacement"
+[workspace]
+resolver = "2"
+members = [
+    "objpoke",
+    "objpoke-cli",
+]
 
-[dependencies]
-anyhow = { version = "1.0.56", features = ["backtrace"] }
-goblin = "=0.4.1"
-scroll = "0.10"
-clap = { version = "4.5.30", features = ["derive"] }
-regex = "1.3.9"
+[workspace.dependencies]
+anyhow = { version = "1.0.96", features = ["backtrace"] }
+goblin = "0.9.3"
```

Comment:
I would need to really test this in depth, there used to some subtle bug in goblin (that's why I pinned `=0.4.1` at the time).

That particular bug has probably been squashed by now but there may be more problems or incompatibilities lurking by suddenly going from 0.4 to 0.9, and I haven't had the time to add a test suite in this pile of very experimental code, I don't feel very confident upgrading such a critical dep without investing a lot more time into testing!

Reply:
Sure, I can revert that, not an issue. I mainly upgraded the dep to that version because armerge also uses that, so I thought you might have just forgotten to upgrade it here.

Reply:
Needed to revert to scroll 0.10 as well, as it didn't work with the newer version